eslint.config.mjs

```diff
@@ -188,88 +169,101 @@ export default tseslint.config([
       ],
       'no-restricted-syntax': ['error', ...NO_RESTRICTED_SYNTAX],
       'no-shadow': 'off',
+      'no-template-curly-in-string': 'error',
+      'no-ternary': 'off',
+      'no-undef': 'off',
+      'no-undef-init': 'off', // TODO: Consider enabling this
+      'no-undefined': 'off',
+      'no-underscore-dangle': 'off',
       'no-unmodified-loop-condition': 'off',
       'no-unneeded-ternary': 'off', // TODO: Consider enabling this
+      'no-unused-vars': 'off',
+      'no-use-before-define': 'off',
       'no-useless-assignment': 'off', // TODO: Consider enabling this
       'no-useless-concat': 'off', // TODO: Consider enabling this
       'no-useless-constructor': 'off',
       'no-useless-return': 'off', // TODO: Consider enabling this
+      'no-warning-comments': 'off',
       'object-shorthand': 'error',
       'one-var': ['off', 'never'], // TODO: Consider enabling this
+      ...Object.fromEntries(
+        Object.keys(perfectionist.rules).map((ruleName) => [
+          'perfectionist/' + ruleName,
+          [
+            [
+              // All disabled rules, can be enabled inline, with e.g. /* eslint perfectionist/sort-objects: error */
```

Comment:
This list of disabled rules is still given the 
```
{ partitionByComment: true, type: 'natural' }
```
config, so that when enabled inline, they match our preferred settings.